modules/accessing-metrics-outside-cluster.adoc

```diff
@@ -49,7 +49,7 @@ $ NAMESPACE=ns1
 +
 [source,terminal]
 ----
-$ curl -H "Authorization: Bearer $TOKEN" -k "https://$HOST/api/v1/query?" --data-urlencode "query=up{namespace='$NAMESPACE'}"
```

Comment:
I think we'll need to think about how to retrieve the needed certs (probably using `oc`) and document that.
Otherwise the `curl` will fail.

Reply:
@machine424 thank you for mentioning it. Can you help me figure out what the steps for this are?

Reply:
yes, I suggest to keep`-k`, otherwise you will get
```
#  curl -H "Authorization: Bearer $TOKEN" "https://$HOST/api/v1/query?" --data-urlencode "query=up{namespace='$NAMESPACE'}"
curl: (60) Peer's certificate issuer has been marked as not trusted by the user.
More details here: http://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
```